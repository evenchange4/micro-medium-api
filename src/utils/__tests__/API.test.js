const API = require('../API');

jest.mock('axios', () => ({
  request: (url, options) => ({ url, options }),
}));

it('should return API getPosts', () => {
  expect(API.getPosts('username', {})).toMatchSnapshot();
});
