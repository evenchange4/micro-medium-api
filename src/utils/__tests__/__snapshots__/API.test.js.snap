// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return API getPosts 1`] = `
Object {
  "options": Object {
    "baseURL": "https://medium.com",
    "method": "get",
    "responseType": "text",
  },
  "url": "@username/latest?format=json",
}
`;
